### DISCUSSION TITLE
bought license, still showing splash screen

### DISCUSSION THREAD

**Discussion by HelmiBastami** [airsdk/Adobe-Runtime-Support#1835] [Q&A] [answered]
I'm about to update an AIR game, so I purchased the HARMAN license so I can include the latest AIR installer without the HARMAN splash screen - but I'm still getting the splash screen.

I'm on Windows 10, using FlashDevelop.

I put the "adt.lic" in the lib folder of the SDK, and have tried 3 different Flex SDK versions. I have internet access and write permissions in the folders. When I try the "adt -license" command suggested in other discussions, it says "License file is present and valid".

I noticed that there's a "adt.cfg" file in the lib folder, and I uncommented the "DebugOut=true" line, however no log file is generated when I publish the project.

I've tried everything I can think of, and exhausted Google search results. I don't have any deep technical knowledge of how these programs work. Any help?

**Comment by ajwfrost** [comment, marked as answer]
Most likely, FlashDevelop isn't using the same SDK location as your AIR SDK with the license/configuration files.. what target are you building for, is this to create a .air file or for a mobile platform?

The first line of the FlashDevelop output when you build should be something like:
```
Running process: C:\Program Files (x86)\FlashDevelop\Tools\fdbuild\fdbuild.exe "[project location and file]" -ipc [guid] -version "[version string]" -compiler "[path to SDK]" -library "C:\Program Files (x86)\FlashDevelop\Library"
```
so can you confirm the `path to SDK` value there is correct?

You can also edit the bat\SetupSDK.bat file to add something like `echo %FLEX_SDK%` under the `validation` section..

thanks

**Reply by HelmiBastami** [reply]
Thank you VERY much for the fast response! That lead to an immediate solution.

The line in the SetupSDK.bat file "FLEX_SDK=" was still set to the wrong SDK location. It was that simple - but I would have never known to look there.
